addons/project/models/project_task.py

```diff
@@ -1720,6 +1720,12 @@ def action_convert_to_subtask(self):
             }
         }
 
+    def action_archive(self):
+        self.ensure_one()
```

Comment:
Is there a reason you've added the self.ensure_one() here ?
The parent method in models.py seems to accept recordset.

Reply:
I can't remember why I did that so I checked and you are correct, this method can indeed accept a recordset :)